cmd/test.go

```diff
@@ -229,6 +239,15 @@ func (t *Test) GetCmd() *cobra.Command {
 				}
 			}
 
+			if buildDelay <= time.Duration(30)*time.Second {
```

Comment:
The same if check should be in the record command as well. And don't mention the example usage of record in the test command. It should be in the record command.